dspace-api/src/main/resources/org/dspace/storage/rdbms/xmlworkflow/postgres/xml_workflow_migration.sql

```diff
@@ -30,8 +30,8 @@ CREATE SEQUENCE cwf_in_progress_user_seq;
 CREATE TABLE cwf_workflowitem
 (
   workflowitem_id integer DEFAULT nextval('cwf_workflowitem_seq') PRIMARY KEY,
-  item_id        INTEGER REFERENCES item(item_id) UNIQUE,
-  collection_id  INTEGER REFERENCES collection(collection_id),
+  item_id        UUID REFERENCES item(uuid) UNIQUE,
+  collection_id  UUID REFERENCES collection(uuid),
```

Comment:
Oooops.... this was from another change I was working on that is completely untested & has nothing to do with the removal of the BundleBItstream object. Reverted this change in a new commit.